app/controllers/items_controller.rb

```diff
@@ -0,0 +1,28 @@
+require 'item_marketplace/presenters/items_index_presenter'
+require 'item_marketplace/presenters/new_item_presenter'
+
+class ItemsController < ApplicationController
+  def index
+    items = RepositoryContainer.get_repo(:item).all
```

Comment:
The get_repo logic is duplicated in 3 places now. In case you change the name of the key, now you will have to remember to update it in all 3 places that you are using this.